packages/wpcom-template-parts/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@automattic/wpcom-template-parts",
-	"version": "1.0.0",
+	"version": "1.0.1",
```

Comment:
Is this really necessary? Has this package ever been released? Looks like it's intended to be private.

Reply:
Good point, reverted 👍